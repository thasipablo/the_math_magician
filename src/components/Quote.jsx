import React, { useEffect, useState } from 'react';

const Quote = () => {
  const [quote, setQuote] = useState('Quote');

  useEffect(() => {
    setQuote('Quote');
  }, []);

  return <div className="quotes">{quote}</div>;
};

export default Quote;
