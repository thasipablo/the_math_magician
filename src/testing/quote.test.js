import { render } from '@testing-library/react';
import Quote from '../components/Quote';

describe('Renders a random computer quote', () => {
  test('render', () => {
    const { quote } = render(<Quote />);
    expect(quote).toMatchSnapshot();
  });
});
