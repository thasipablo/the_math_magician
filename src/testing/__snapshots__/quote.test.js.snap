// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders a random computer quote render 1`] = `undefined`;
